src/main/java/mate/academy/security/jwt/JwtTokenProvider.java

```diff
@@ -57,13 +65,12 @@ private String getUsername(String token) {
                 .parseClaimsJws(token)
                 .getBody()
                 .getSubject();
-
     }
 
     public String resolveToken(HttpServletRequest req) {
         String bearerToken = req.getHeader("Authorization");
         if (bearerToken != null && bearerToken.startsWith("Bearer ")) {
-            return bearerToken.substring(7);
+            return bearerToken.substring(TOKEN_PAYLOAD_POSITION);
```

Comment:
I would rather make "Bearer " a constant, and 7 would be TOKEN_PREFIX.length(), that would make it immediately clear how they are connected

Reply:
ok